### PR TITLE
docs: Update JavaScript WDIO driver touch action.

### PR DESCRIPTION
**Pull request by blane1988**
## Proposed changes

For tapping on UI elements, older documentation had an action called `touchPerform` which is no longer recognized by Appium. The action `touchAction` works.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: commands-yml/commands/interactions/touch/tap.yml [hunk 1]
```diff
@@ -26,12 +26,7 @@ example_usage:
       await action.perform();
   javascript_wdio:
     |
-      driver.touchPerform({
-        action: 'tap',
-        options: {
-          element: element
-        }
-      });
+      <element>.touchAction('tap');
```
Comment (KazuCocoa) [inline commands-yml/commands/interactions/touch/tap.yml]: Could you remain driver.touchAction, too?
https://webdriver.io/docs/api/browser/touchAction.html

Comment (blane1988) [reply commands-yml/commands/interactions/touch/tap.yml]: Changed `<element>` to `driver`. Thanks.

**Review by KazuCocoa — APPROVED** [review]

**Comment by blane1988** [comment]
How long does it take for a merge to be reflected in the Appium documentation? When I go to the doc page for touchAction, it still shows the old method.

**Comment by KazuCocoa** [comment]
Weekly auto-generation has been finished https://github.com/appium/appium/pull/13899
Perhaps we should update https://github.com/appium/appium.io

**Pull request merged**